common/Resources/ti.internal/extensions/node/stream.js

```diff
@@ -0,0 +1,124 @@
+import EventEmitter from './events';
+import util from './util';
+
+function Stream(opts) {
```

Comment:
<!--
  0 failure: 
  1 warning:  common/Resources/...
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/extensions/node/stream.js;
  Line: 4;
-->

- :warning: common/Resources/ti.internal/extensions/node/stream.js line 4 – 'opts' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)